storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoUtils.java

```diff
@@ -49,22 +59,38 @@ public static ResourceDao sampleResourceDao() throws InvalidIssnException, Malfo
         Item item = attempt(() -> Item.fromJSON(objectMapper.writeValueAsString(element))).orElseThrow();
         return ItemUtils.toAttributeValues(item);
     }
-
+    
     public static Stream<Dao<?>> instanceProvider() throws InvalidIssnException, MalformedURLException {
         ResourceDao resourceDao = sampleResourceDao();
         DoiRequestDao doiRequestDao = doiRequestDao(resourceDao.getData());
-        return Stream.of(resourceDao, doiRequestDao);
+        MessageDao messageDao = sampleMessageDao();
```

Comment:
This method provides Dao instances in the DaoTest class so that the Dao tests are run for all children of Dao